Lib/base64.py

```diff
@@ -576,6 +576,27 @@ def decodebytes(s):
     return binascii.a2b_base64(s)
 
 
+# Use accelerated implementations of originally pure-Python parts if possible.
+try:
+    from _base64 import (_a85encode, _a85decode, _b85encode,
+                         _b85decode, _z85encode, _z85decode)
```

Comment:
Given these are already in a private module, you can remove the prefix. That means the `_copy_attributes` function only needs to copy `__doc__`, and `__module__` can be set to the static `'base64'`.

Reply:
Done.